### PR TITLE
+geoDistanceRange DSL function 

### PR DESCRIPTION
**Pull request by j33f**
Adds geoDistanceRange DSL function 
+ related tests and doc +change Kuzzle version in package.json

### CODE REVIEW

File: lib/api/dsl/methods.js [hunk 1]
```diff
@@ -751,11 +747,93 @@ module.exports = methods = {
...
+    fieldName = Object.keys(filter)[0];
+    if (fieldName === 'from' || fieldName === 'to') {
+      fieldName = Object.keys(filter)[1];
+    }
+    if (fieldName === 'from' || fieldName === 'to') {
+      fieldName = Object.keys(filter)[2];
+    }
 
-    deferred.reject(new KuzzleError('geoDistanceRange is not implemented yet.'));
+    geoFilter = filter[fieldName];
```
Comment (scottinet) [inline lib/api/dsl/methods.js]: As it is, ``fieldName`` may be undefined, leading to a ``TypeError`` when initializing ``geoFilter``.

For instance with this filter:

```json
{ "from": "foobar" }
```

Comment (j33f) [inline lib/api/dsl/methods.js]: yep! corrected

**Comment by ballinette** [comment]
+1

**Comment by benoitvidis** [comment]
+1

**Pull request merged**